### PR TITLE
Update LoginApp.css

### PR DESCRIPTION
**Pull request by coxwm11**
Changes background in css

Screen Shot 2021-02-15 at 12 21 08 AM

### CODE REVIEW

File: src/Components/LoginApp.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
 }
 
 .background_login {
-  background-image: url(/Recess_Bg.jpg);
+  background-image: url(/Recess_Bg_2.jpg);
```
Comment (msknobloch2) [inline src/Components/LoginApp.css:6]: This jpg does not seem to exist in this pull request.  Can you add it in?

Comment (jonathanmcdaniel) [reply src/Components/LoginApp.css:6]: added

**Review by msknobloch2 — APPROVED** [review]

**Review by jonathanmcdaniel — APPROVED** [review]
Looks great Wendy! Love the background!

**Comment by msknobloch2** [comment]
closes #118

**Review by gateship1 — APPROVED** [review]
this looks amazing!

**Pull request merged**